airbyte-integrations/connectors/source-prestashop/source_prestashop/prestashop.yaml

```diff
@@ -0,0 +1,469 @@
+version: "0.3.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["{{ options['data_key'] }}"]
+  requester:
+    url_base: "{{ config['url'] }}/api/"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['access_key'] }}"
+    request_options_provider:
+      request_headers:
+        Output-Format: JSON
+      request_parameters:
+        display: full
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      class_name: source_prestashop.components.LimitOffsetPaginator
+      pagination_strategy:
+        type: "OffsetIncrement"
+        page_size: 50
+      limit_offset_option:
+        inject_into: "request_parameter"
+        field_name: "limit"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    primary_key: "id"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  base_incremental_stream:
+    $ref: "*ref(definitions.base_stream)"
+    stream_cursor_field: "date_upd"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      stream_slicer:
+        class_name: source_prestashop.components.NoSlicer
+        primary_key: "*ref(definitions.base_stream.primary_key)"
+  addresses_stream:
+    $ref: "*ref(definitions.base_incremental_stream)"
+    $options:
+      name: "addresses"
+      path: "/addresses"
```

Comment:
if I'm not mistaken, slash may be omitted

Reply:
I did not checked, but a lot of connectors use this slash in path
and as for me it's more expressively that it's "path" :)